src/test/java/hudson/plugins/git/util/DefaultBuildChooserTest.java

```diff
@@ -30,6 +44,46 @@ public void testChooseGitRevisionToBuildByShaHash() throws Exception {
         candidateRevisions = buildChooser.getCandidateRevisions(false, "aaa" + shaHashCommit1.substring(3), testGitClient, null, null, null);
         assertTrue(candidateRevisions.isEmpty());
     }
+
+    @Test
+    public void testChooseWithMultipleTag() throws Exception {
```

Comment:
javadoc description will be appreciated because it can be difficult to follow executed commands